package.json

```diff
@@ -52,7 +53,9 @@
   "dependencies": {
     "@xmldom/xmldom": "^0.7.5"
   },
-  "devDependencies": {},
+  "devDependencies": {
```

Comment:
anyway to get rid of this dev dependency?

Reply:
Those are used by the EventEmitter Node interface.
It's not code but just typings for Node. Those are stripped out once converted in JS.
It won't pollute anything. If you don't use TS it will be ignored.